.github/workflows/nodejs.condorb.test.yml

```
@@ -0,0 +1,138 @@
+# This workflow will do a clean installation of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: RUN CONDO RB SERVICE TESTS
+
+on:
+  workflow_dispatch:
+    branches:
+      - '*'
+  push:
+    branches: [ master ]
+    paths:
+      - 'apps/condorb'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'apps/condorb'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 16.x ]
+        database: [ "postgresql://postgres:postgres@127.0.0.1/main" ]
+
+    steps:
+      - name: Checkout code with submodules
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: recursive
+          ssh-key: ${{ secrets.SSH_DOCK_SERVER_PRIVATE_KEY }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+          cache-dependency-path: yarn.lock
+
+      - name: Docker compose up databases
+        run: |
+          cp .env.example .env
+          docker-compose up -d postgresdb redis
```

It'd be better to see new syntax here
```suggestion
          docker compose up -d postgresdb redis
```